packages/lodestar/test/sim/singleNodeSingleThread.test.ts

```diff
@@ -13,14 +13,15 @@ import {sleep, TimestampFormatCode} from "@chainsafe/lodestar-utils";
 /* eslint-disable no-console, @typescript-eslint/naming-convention */
 
 describe("Run single node single thread interop validators (no eth1) until checkpoint", function () {
-  const timeout = 120 * 1000;
   const testParams: Pick<IBeaconParams, "SECONDS_PER_SLOT" | "SLOTS_PER_EPOCH"> = {
     SECONDS_PER_SLOT: 2,
     SLOTS_PER_EPOCH: 8,
   };
   const manyValidatorParams: Partial<IBeaconParams> = {
     ...testParams,
-    TARGET_AGGREGATORS_PER_COMMITTEE: 1,
+    // Reduce from 16 of minimal but ensure there's almost always an aggregator per committee
```

Comment:
what does "Reduce from 16 of minimal" mean?

Reply:
The minimal config has 16 as value